portfolio/src/main/webapp/script.js

```diff
@@ -42,6 +42,8 @@ function getJsonMessage() {
   // The fetch() function returns a Promise because the request is asynchronous.
   const responsePromise = fetch('/data');
 
+  console.log(responsePromise);
```

Comment:
Probably can remove this before submitting, along with other log statements below :)